src/app/locales/en.json

```diff
@@ -203,6 +203,10 @@
       "zero": "No Responses",
       "one": "%(count)s Response",
       "other": "%(count)s Responses"
+    },
+    "post_key_warning": {
```

Comment:
language-wise we should make this warning a lot louder. maybe:

```json
"post_key_warning": {
  "confirm": "You are about to publish a STEEM private key or master password. You will probably lose control of the associated account and all its funds.",
  "warning": "Legitimate actors, including employees of Steemit Inc., will never ask you for a private key or master password."
}
```

Reply:
Agree, proposed "control of the associated account" is much better and more accurate than "control of your account and funds"